src/controllers/users_controller.ts

```diff
@@ -1,14 +1,33 @@
-import crypto from 'crypto';
 import util from '../utils/authUtil';
+import { jwt } from '../utils';
+import crypto from 'crypto';
 import resMessage from '../utils/resMessage';
 import express, { Request, Response, NextFunction, Send } from 'express';
```

Comment:
```suggestion
import { Request, Response } from 'express';
```
Send랑 NextFunction 안쓰면 지우는게 좋을 듯! express도!